src/main/resources/com/cloudbees/jenkins/plugins/bitbucket/BitbucketSCMNavigator/config.jelly

```diff
@@ -1,30 +1,25 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form" xmlns:c="/lib/credentials">
-    <f:entry title="${%Owner}" field="repoOwner">
-        <f:textbox/>
-    </f:entry>
-    <f:entry title="${%Scan credentials}" field="credentialsId">
-        <c:select/>
-    </f:entry>
-    <f:entry title="${%Repository name pattern}" field="pattern">
-        <f:textbox default=".*"/>
-    </f:entry>
-    <f:entry field="autoRegisterHooks">
-        <f:checkbox title="${%Auto-register webhooks}" />
-    </f:entry>
-    <f:advanced>
-        <f:entry title="${%Include branches}" field="includes">
-            <f:textbox default="*"/>
-        </f:entry>
-        <f:entry title="${%Exclude branches}" field="excludes">
-            <f:textbox/>
-        </f:entry>
-        <f:entry title="${%Checkout credentials}" field="checkoutCredentialsId">
-            <c:select default="${descriptor.SAME}"/>
-        </f:entry>
-        <f:entry title="${%Bitbucket Server URL}" field="bitbucketServerUrl">
-          <f:textbox />
-        </f:entry>
-    </f:advanced>
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form" xmlns:c="/lib/credentials" xmlns:scm="/jenkins/scm/api/form">
+  <j:choose>
+    <j:when test="${descriptor.serverUrlSelectable}">
```

Comment:
Something seem wrong here.
We don't get the the drop down and/or unable to provide the Bitbicket Server url.